js_modules/dagster-ui/packages/ui-core/src/ticks/DryRunRequestTable.tsx

```diff
@@ -80,7 +78,7 @@ export const RunRequestTable: React.FC<Props> = ({
   );
   return (
     <div>
-      <Table style={{borderRight: `1px solid ${Colors.KeylineGray}`}}>
+      <Table style={{borderRight: `1px solid ${Colors.KeylineGray}`, tableLayout: 'fixed'}}>
```

Comment:
Is this also necessary? This causes the table columns to be defined by the content widths of the first row only, so if the first row has no tags, the Tags column will be super small.

Reply:
I believe it is. The table width has a width of 100% though